internal/signaling/stores/postgres.go

```diff
@@ -154,10 +154,10 @@ func (s *PostgresStore) CreateLobby(ctx context.Context, game, lobbyCode, peerID
 	}
 	now := util.Now(ctx)
 	res, err := s.DB.Exec(ctx, `
-		INSERT INTO lobbies (code, game, public, created_at, updated_at)
-		VALUES ($1, $2, $3, $4, $4)
+		INSERT INTO lobbies (code, game, public, meta, created_at, updated_at)
+		VALUES ($1, $2, $3, $4, $5, $5)
 		ON CONFLICT DO NOTHING
-	`, lobbyCode, game, public, now)
+	`, lobbyCode, game, public, customData, now)
```

Comment:
This serializes correctly from map to jsonb automatically? Nice!

Reply:
Yes it seems. We did the opposite (Scan) in other places already.